doc/sphinx/source/input.rst

```diff
@@ -311,8 +311,10 @@ A list of the datasets for which a CMORizers is available is provided in the fol
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
 | ESACCI-AEROSOL               | abs550aer, od550aer, od550aerStderr, od550lt1aer, od870aer, od870aerStderr (aero)                    |   2  | NCL             |
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
+| ESACCI-BIOMASS               | agb (Lyr, frequency=yr)                                                                              |   2  | Python          |
++------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
 | ESACCI-CLOUD                 | clivi, clt, cltStderr, clwvi, lwp, rlut, rlutcs, rsut, rsutcs, rsdt, rlus, rsus, rsuscs (Amon),      |   2  | Python          |
-|                              | clt, clwvi, cod (day)                                                                                |   2  |                 |
+|                              | clt, clwvi, cod (day)                                                                                |      |                 |
```

Comment:
Did you remove the tier here on purpose?

Reply:
I thought tier applies to the whole dataset, so I found it confusing to have two tier definitions and removed the second one (acknowledging that one was probably for the monthly data and the other one for the daily data).